src/promptflow-evals/promptflow/evals/synthetic/README.md

```diff
@@ -230,11 +230,11 @@ This should result in something like:

```

Comment:
why are we adding this file?

Reply:
That's not a change, it's just right above my changes, and git's file view doesn't know how to handle extremely long single lines.